mozjs-sys/build.rs

```diff
@@ -3,12 +3,16 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use bindgen::Formatter;
+use flate2::read::GzDecoder;
+use flate2::write::GzEncoder;
+use flate2::Compression;
 use std::env;
 use std::ffi::{OsStr, OsString};
-use std::fs;
+use std::fs::{self, File};
```

Comment:
Although there isn't a consistent import style yet, this should probably be changed.
```suggestion
use std::fs;
use std::fs::File;
```